pom.xml

```diff
@@ -42,7 +42,7 @@
         <cxf.compiler.fork>false</cxf.compiler.fork>
         <cxf.build-utils.version>3.4.5-SNAPSHOT</cxf.build-utils.version>
         <cxf.xjc-utils.version>4.0.0-SNAPSHOT</cxf.xjc-utils.version>
-        <cxf.jdk.version>17</cxf.jdk.version>
+        <cxf.jdk.version>11</cxf.jdk.version>
```

Comment:
@jimma the tests are failing for me

```
Caused by: java.lang.UnsupportedClassVersionError: org/springframework/test/context/junit/jupiter/SpringExtension has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1016)
```

We need to revert it back to `17` :(

Reply:
Which test is failing ? I use this jdk version : 

```
openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment Homebrew (build 17.0.1+1)
OpenJDK 64-Bit Server VM Homebrew (build 17.0.1+1, mixed mode, sharing)
```

Reply:
That's the thing, we always need JDK-17 to build, why do we need to use `11` in `pom.xml`?

Reply:
The only benefit I see is to stick to JDK-11 source / target, but we cannot build on this JDK-11, only on JDK-17, we could probably use enforcer plugin for that.

Reply:
> The only benefit I see is to stick to JDK-11 source / target, but we cannot build on this JDK-11, only on JDK-17, we could probably use enforcer plugin for that.

The big benefit I see now is  we can still produce the JDK-11 artifacts. There are still a lot of projects with JDK-11 which doesn't use spring and only consume several cxf artifacts.

Reply:
> That's the thing, we always need JDK-17 to build, why do we need to use `11` in `pom.xml`?
We use JDK-17 to build and we can still build cxf for JDK-11 backward compatibility when we set this 11 target flag.

Reply:
Only when we get rid of any Spring deps, right?

Reply:
Yes, this could help us to decouple Spring in the future and also useful for the project has the CXF dependencies with Spring exclusion and still using JDK-11.

Reply:
:+1: